### DISCUSSION TITLE
part-ii/2019-141/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#272] [Question comments]
Part II 2019 Statistical Physics: Paper 3, Section II, D 

https://questions.tripos.org/part-ii/2019-141/

**Comment by thenewtonium** [comment]
<details><summary>Gibbs distribution</summary>

This is the distribution of states in the grand canonical ensemble. Derivation is similar to the Boltzmann distribution (canonical ensemble), except that particles *as well as* heat can be transferred between the "reservoir" and the system, and the reservoir is at fixed chemical potential $\mu$ *as well as* fixed temperature $T$.
</details>

Finding the mean occupation number $n_F (\varepsilon)$ is just stuff from notes.

<details><summary>Showing density of states is indept. of ε</summary>

This was on the second examples sheet.

We know that the energy levels of free particles in a box in 2d are $C_x n_x ^2 + C_y n_y ^2$ for $(n_x,n_y) \in \mathbb{N}^2$, and some constants $C_x$, $C_y$ whose values will not matter for the purposes of this argument.

Define $G(\varepsilon) := \#${states with energy $\leq \varepsilon$}. Rescaling our axes as $x = n_x \sqrt{C_x}$, $y = n_y \sqrt{C_y}$, we can approximate $G(\varepsilon) \approx$ Area$\{(n_x, n_y)\in \mathbb{R}_{\geq 0}^2 | C_x n_x^2 + C_y n_y ^2 \leq \varepsilon\}$  = $\frac{1}{4\sqrt{C_x C_y}}$Area{$(x,y)\in \mathbb{R}^2 | x^2 + y^2 \leq \varepsilon$} $\propto \epsilon$. Thus $g(\varepsilon) = G'(\varepsilon)$ is constant in $\varepsilon$.
</details>

<details><summary>Mean number of particles</summary>

To get this, use the definition of $\varepsilon_F$ and rearrange to obtain that $g(\varepsilon) = N/\varepsilon_F$, and use that $\mu \approx \varepsilon_F$ for $T << \varepsilon_F$.

</details>

<details><summary>Small T heat capacity</summary>

Use the prev. part to write $\left\langle E \right\rangle \approx \frac{N}{\varepsilon_{F}}\int_0 ^\infty \frac{\varepsilon d \varepsilon}{e^{\left(\varepsilon-\varepsilon_{F}\right) / k_B T}+1}$, and sub in $x := \left(\varepsilon-\varepsilon_{F}\right) / k_B T$ to get $\left\langle E \right\rangle \approx \frac{N}{\varepsilon_{F}}\int_{-\varepsilon_F/k_B T} ^\infty \frac{(x + (\varepsilon_F/k_B T)) dx}{e^{x}+1}$.

Use that $k_B T<<\varepsilon_F$ is large to replace the lower limit by $-\infty$ then differentiate w.r.t. $T$ to obtain
$C_V \approx \frac{N}{k_B T^2}\int_{-\infty} ^\infty \frac{dx}{e^{x}+1}$, i.e. the power law is $C_V \sim T^{-2}$.

---

Alternatively, differentiate $\left\langle E \right\rangle$ in integral form first, then make the $x := \left(\varepsilon-\varepsilon_{F}\right) / k_B T$ substitution and "blow up" the lower limit, and notice the odd term of the integrand vanishes leaving a power law.
</details>
